tests/test_api_result.py

```diff
@@ -1927,6 +1928,24 @@ def urllib_mock() -> UrllibMocker:
         yield mocker
 
 
+class UrllibAndRequestMocker:
```

Comment:
@bossie Moved the double mock to here. It is almost a drop-in-replace for the original mocks. Ok?